docs/Getting-Started/Starting-Pantheon.md

```diff
@@ -67,63 +69,66 @@ call [JSON-RPC API methods](../Reference/JSON-RPC-API-Methods.md) to confirm the
 To run a node that mines blocks at a rate suitable for testing purposes: 
 
 ```bash
-pantheon --dev-mode --network-id="2018" --bootnodes --miner-enabled --miner-coinbase=0xfe3b557e8fb62b89f4916b721be55ceb828dbd73 --rpc-cors-origins="all" --host-whitelist="all" --ws-enabled --rpc-enabled --datadir=/tmp/tmpDatdir
+pantheon --network=dev --miner-enabled --miner-coinbase=0xfe3b557e8fb62b89f4916b721be55ceb828dbd73 --rpc-http-cors-origins="all" --host-whitelist="all" --rpc-rpc-ws-enabled --rpc-http-enabled --data-path=/tmp/tmpDatdir
 ```
 
-Alternatively, use the following [configuration file](../Configuring-Pantheon/Using-Configuration-File.md) and `--bootnodes` on the command line to start a node with the same options as above: 
+Alternatively, use the following [configuration file](../Configuring-Pantheon/Using-Configuration-File.md) 
+on the command line to start a node with the same options as above: 
 ```toml
-dev-mode=true
-network-id="2018"
+network="dev"
 miner-enabled=true
 miner-coinbase="0xfe3b557e8fb62b89f4916b721be55ceb828dbd73"
 rpc-cors-origins=["all"]
 host-whitelist=["all"]
-ws-enabled=true
-rpc-enabled=true
-datadir="/tmp/tmpDatadir"
+rpc-ws-enabled=true
+rpc-http-enabled=true
+data-path="/tmp/tmpdata-path"
 ```
 
-## Run a Node on Ropsten Testnet 
-
 !!!note
-    From v0.8.2, use the [`--ropsten` option](../Reference/Pantheon-CLI-Syntax.md#options) 
-    instead of the following options. For v0.8.1, use the following options.
+    `--network` option is new from 0.9
```

Comment:
I don't think we need this - so many of the command line options have changed in 0.9 we'll have a general 'Breaking Changes to the Command Line in 0.9" note in the doc and release notes